### ISSUE TITLE
POWER BI Report not refresh the data after the data set refresh.

### ISSUE THREAD

**Issue by rahulpawar16** [akshay5995/powerbi-report-component#183]
Sorry to miss info about the previous question. 

Stage first-> I render report using the powerbi-report-component, I provided the accessToken, embedUrl, reportId, etc. It is showing report perfectly. fine till now,

Stage Second-> After doing some changes in MSSQL DB. I am Refreshing the PowerBI DataSet report using the DatasetId and GroupId. from the frontend. after 3 second my dataset refreshed, In PowerBi it is affecting. but after rerendering the powerbi-report-component using the shouldComponentUpdate I changes the props and it's rerendering to but not reflecting the actual dataset refresh, and not showing on frontend.

**Comment by akshay5995** [comment]
@rahulpawar16 Can you rephrase the question. It's not very clear. Thanks.

**Comment by akshay5995** [comment]
Thanks for the explanation. Maybe you can try something like  [this](https://github.com/microsoft/PowerBI-JavaScript/blob/b5caee0ee077d1a136961fa4506108aeaba8de57/demo/app/index.js#L37)

Get the report object from `onLoad` function and ```report.reload().catch(err => console.log(err))```

**Comment by akshay5995** [comment]
@rahulpawar16 Can you confirm if it's working so we can close the issue ?

**Comment by akshay5995** [comment]
Closing due to no activity
